server.js

```diff
@@ -19,4 +20,6 @@ app.post('/api/foods', FoodsController.create);
 app.patch('/api/foods/:id', FoodsController.update);
 app.delete('/api/foods/:id', FoodsController.destroy);
 
+app.get('/api/meals/:category/:year/:month/:day', MealsController.index);
```

Comment:
Wondering if you think passing through ?date= might be cleaner? This just seems like a long route.